src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -542,7 +628,7 @@ partial class BoundDup
 
     partial class BoundBaseReference : IInstanceReferenceExpression
     {
-        bool IInstanceReferenceExpression.IsExplicit => true;
+        bool IInstanceReferenceExpression.IsExplicit => this.Syntax.Kind() == SyntaxKind.BaseExpression;
```

Comment:
Is it possible to have a BoundBase that didn't have "base." In the original code?